### PR TITLE
Feature/dev client 99

### PR DESCRIPTION
**Pull request by MikeMcGarry**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +32,24 @@ or with docker using
 $ docker run -it --rm -v $(pwd):/usr/src/lusid-client-java -w /usr/src/lusid-client-java maven:3.6.0-jdk-11-slim mvn -e -fae -Dtest=ConnectivityTests test
 ```
 
-# API Credentials
+# Using a Proxy
 
-All authenticated calls to the LUSID API require an OpenID Connect ID token which is issued from a your token issuer url. The details of these can be found on your LUSID portal under "Applications" within the "Identity and Access Management" section. 
+If the connectivity tests fail and/or you need to use a proxy, remove the "@Ignore" annotation on the
```
Comment (rizwansaeed) [inline README.md]: make the `@Ignore` markdown

File: README.md [hunk 2]
```diff
@@ -20,13 +32,24 @@ or with docker using
...
+verify_connection_with_proxy() test and run it adding in your:
```
Comment (rizwansaeed) [inline README.md]: as above

File: src/test/java/com/finbourne/lusid/integration/ConnectivityTests.java [hunk 3]
```diff
@@ -63,13 +63,15 @@ public void verify_connection() throws IOException {
...
-        //  this should be your proxy address
-        InetSocketAddress   proxy = new InetSocketAddress(InetAddress.getLocalHost(), 8889);
-
         //  username and password for the proxy
+        InetAddress  proxyAddress = InetAddress.getLocalHost();
+        Integer port = 8889;
         String  proxyUsername = "user";
         String  proxyPassword = "password";
 
+        //  this should be your proxy address
```
Comment (rizwansaeed) [inline src/test/java/com/finbourne/lusid/integration/ConnectivityTests.java]: this comment should be above, `InetSocketAddress` is an implementation detail

File: README.md (hunk 1 above)
Comment (MikeMcGarry) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (MikeMcGarry) [reply README.md]: Done

**Pull request merged**